docs/source/examples/petstore/petstore.all.rego

```diff
@@ -22,7 +22,7 @@ deny {
 	seal_list_contains(seal_subject.groups, `everyone`)
 	input.verb == `buy`
 	re_match(`petstore.pet`, input.type)
-	input.age <= 2
+	input.ctx[i].age <= 2
```

Comment:
isn't `some i` missing before this line?

Reply:
and also from all other rego stanzas with `ctx[i]`?

Reply:
you're about input.type and input.verb ?
it also should be multiplied?

Reply:
Seems implemented